samplejava/build.gradle

```diff
@@ -27,18 +27,21 @@ android {
         targetCompatibility JavaVersion.VERSION_1_8
     }
 
+    buildFeatures {
+        viewBinding true
+    }
+
     packagingOptions {
         exclude 'META-INF/*.kotlin_module'
     }
 }
 
 dependencies {
-    implementation "org.jetbrains.kotlin:kotlin-stdlib:1.4.10"
-    implementation "io.getstream:stream-chat-android:4.4.5"
-    implementation "io.coil-kt:coil:1.0.0"
+    implementation "io.getstream:stream-chat-android-ui-components:4.5.3"
+    implementation 'io.coil-kt:coil:1.1.1'
```

Comment:
Is Coil dependency needed here?
Our SDK already ask for it

Reply:
Not sure. It was in the last tutorial - I'll remove it and test.

Reply:
If our SDK provides it as an api dependency we should be able to use it.

Reply:
Ok, I didn't see it is needed inside the tutorial.
We shouldn't provide it as API, so it is fine as it is ;)